build.gradle

```diff
@@ -5,3 +5,22 @@ plugins {
 }
 
 version = findProperty("LIBRARY_VERSION")
+
+def getValueFromPropertiesFile = { propFile, key ->
+    if (!propFile.isFile() || !propFile.canRead())
+        return null
+    def prop = new Properties()
+    def reader = propFile.newReader()
+    try {
+        prop.load(reader)
+    } finally {
+        reader.close()
+    }
+    return prop.get(key)
+}
+
+ext.getProperty = { name, defValue ->
```

Comment:
I know this was just moved around from a different gradle script, but I'm not convinced that having two similarly worded methods `findProperty` and `getProperty` are the right way to go. 
I think using a more specific name like `getValueFromLocalProperties` might be better.

Reply:
I like that idea too, I'll update the PR.